### ISSUE TITLE
Overriding check config from plan

### ISSUE THREAD

**Issue by pun-ky** [adamcin/oakpal#46]
is there any way to override particular check config via plan JSON?

e.g I want to use oakpal basic checklist with one exception, with allowing subpackages inside.
how to do that?

 I mean... http://adamcin.net/oakpal/oakpal-core/the-basic-checklist.html

**Comment by adamcin** [comment]
@pun-ky yes, it should be possible to override specific configs. You should define a child of `.checks[]` with a `.name` property that matches the `[[{moduleName}/]{checklist}/]{checkName}` of the check you wish to override, an empty `.impl` property, and a `.config` object with key/value pairs to add/override wrt to the overridden check config.

Per your example:
```json
    {
      "name": "basic/subpackages",
      "config": {
        "denyAll": false
      }
```
You can also use the override to completely skip the check:
```json
    {
      "name": "basic/subpackages",
      "skip": true
    }
```

**Comment by pun-ky** [comment]
Cool. Thanks for explanation. Is such info available in docs?
